modules/update-upgrading-oc-adm-upgrade-status.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-status_{context}"]
+= Retrieving information about a cluster update using oc adm upgrade status (Technology Preview)
+
+When updating your cluster, it is useful to understand how your update is progressing. While the `oc adm upgrade` command returns limited information about the status of your update, this release introduces the `oc adm upgrade status` command as a Technology Preview feature. This command decouples status information from the `oc adm upgrade` command and provides specific information regarding a cluster update, including the status of the control plane and worker node updates.
+
+The `oc adm upgrade status` command is read-only and will never alter any state in your cluster. 
+
+[IMPORTANT]
+====	
+The `oc adm upgrade status` command is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features - Scope of Support].
+====
+
+The `oc adm upgrade status` command can be used for clusters from version 4.12 up to the latest supported release. 
+
+[IMPORTANT]
+====
+While your cluster does not need to be a TechPreview-enabled cluster, you must enable the `oc adm upgrade status` Technology Preview environment variable, otherwise the `oc` will not recognize the command and you will not be able to use the feature. 
```

Comment:
```suggestion
While your cluster does not need to be a Technology Preview-enabled cluster, you must enable the `oc adm upgrade status` Technology Preview environment variable, otherwise the `oc` will not recognize the command and you will not be able to use the feature. 
```